modules/core/src/main/java/org/apache/synapse/mediators/db/DBLookupMediator.java

```diff
@@ -112,6 +113,11 @@ protected void processStatement(Statement stmnt, MessageContext msgCtx) {
                     rs.close();
                 } catch (SQLException e) {}
             }
+            if (ps != null) {
+                try {
+                    ps.close();
+                } catch (SQLException ignore) {}
```

Comment:
@madhawa-gunasekara Good point, but all the other similar close statements are ignoring the exception, hence ignored to have a consistence. IMO, if we added a warning we may need to add to the other close statements as well.